app-server/migrations/src_migrations/main/scala/11.scala

```diff
@@ -0,0 +1,17 @@
+import slick.driver.PostgresDriver.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M11 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(11)(List(
+    sqlu"""
+CREATE EXTENSION postgis;
+CREATE TABLE footprints (
+  id UUID PRIMARY KEY NOT NULL,
+  organization_id UUID REFERENCES organizations(id) NOT NULL,
+  created_at TIMESTAMP NOT NULL,
+  modified_at TIMESTAMP NOT NULL,
+  polygon geometry(MULTIPOLYGON, 4326) NOT NULL
```

Comment:
If this is a multipolygon it should be called multipolygon.

Also, I think we should probably store this in web mercator to simplify spatial queries that are going to usually be specified in meters and not in terms of decimal degrees (e.g. all polygons within X meters of a point)